notebooks/NIRSPEC/IFU/JWPipeNB-NIRSpec-IFU.ipynb

```diff
@@ -26,7 +26,7 @@
    },
```

Comment:
<p>If you go with the changes to simplify get_matching, I don't think we should need to load these into numpy arrays.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwst-pipeline-notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Yep; I've reverted these changes.</p>